src/test/java/org/opensearch/security/dlic/rest/api/RoleBasedAccessTest.java

```diff
@@ -173,11 +177,9 @@ public void testActionGroupsApi() throws Exception {
         // Worf, has access to roles API, get captains role
         response = rh.executeGetRequest(ENDPOINT + "/roles/opendistro_security_role_starfleet_captains", encodeBasicHeader("worf", "worf"));
         Assert.assertEquals(HttpStatus.SC_OK, response.getStatusCode());
-        Assert.assertEquals(
-            new SecurityJsonNode(DefaultObjectMapper.readTree(response.getBody())).getDotted(
-                "opendistro_security_role_starfleet_captains.cluster_permissions"
-            ).get(0).asString(),
-            "cluster:monitor*"
+        assertThat(
```

Comment:
Props to @willyborankin who figured this out so quickly I didn't even get a chance to look into it.

Upgraded this test to make sure both permissions are present instead of only the first one.